app/services/claims/injection_channel.rb

```diff
@@ -0,0 +1,8 @@
+module Claims
+  class InjectionChannel
+    def self.for(claim)
+      return 'cccd_development' if claim.nil?
+      claim.agfs? ? Settings.slack.channel : 'cccd_cclf_injection'
```

Comment:
As discussed, can we just put the config for the channels in the app entirely. clear up the TD env vars later